echopype/convert/parse_azfp.py

```diff
@@ -129,10 +129,10 @@ def load_AZFP_xml(self):
 
         def get_value_by_tag_name(tag_name, element=0):
             """Returns the value in an XML tag given the tag name and the number of occurrences."""
-            return px.getElementsByTagName(tag_name)[element].childNodes[0].data
+            return root.iter(tag_name).__next__().text
```

Comment:
This new statement doesn't make use of the `element` argument, which is required by the call in line 146.

Reply:
@emiliom I removed the usage of this function for parsing `AZFP` XML files.